### DISCUSSION TITLE
What career route is best for my goals?

### DISCUSSION THREAD

**Discussion by Mr3bzz** [community/community#84768] [Other Feature Feedback, Questions, & Ideas]
I've always aspired to be a developer and have always been facinated with computing / programming. 

I currently work within a growing software company.. I am CSO, so knowing the systems is my role. However we have in house devs using c# and .net ..

And live services using MYSQL.. this would be the first position I make a transition too, as I'm somewhat familiar with MySQL syntax.. 

However I guess what my question is .. do you think it's wise for me to learn C# and .net? Is it in demand? Widely used? Still valuable ? Highly paid? 

Also what's your advice on where to start? And specific recommendations to courses out there on udemy? YouTube? 

Thanks for any advice and help.

**Comment by MrPurples** [comment]
Yes it is worth it . You wont regret it .

**Comment by h3ct0rjs** [comment]
Hey @Mr3bzz hope you're doing great, 

Well, there is currently a great website https://roadmap.sh/ where you will be able to check a few roadmaps. I would say that try to explore a bit, since you know the business goals and main objectives I would highly recommend looking for a future in the current company or a new company, with this at least you will be able to experiment in the transition. 

Technical details can be learned with discipline, patience, and time, but knowing important aspects of the business is something related to experience in the field.  Also getting an expert at something requires strong dedication and time. 

Hope this answer helps you a lot. 

Good Luck

**Comment by naveen-98** [comment]
Hello @Mr3bzz 

Learning C# and .NET is a wise move, especially since your company uses these technologies. They are in-demand, versatile, and can lead to well-paid positions. Start with online courses on platforms like Udemy and practice through projects. Stay updated on the latest versions, and consider books and documentation for a comprehensive understanding. Good luck!

**Reply by Mr3bzz** [reply]
I've began my journey to Dev 💻.

Currently loving C#, haven't yet began learning .NET as I believe knowing C# before the framework makes sense.. 

Thanks for the tips and advice.

Always open to more tips and advice haha 😀
